GitHub CI failing

GitHub Continuous Integration (CI) unit test setup is failing to install mapbox due to missing `GDAL_CONFIG` or `GDAL_VERSION`.

#### Code Sample, a minimal, complete, and verifiable piece of code

I don't have a code sample, but the problem can be seen in the GitHut CI for #14, #15, and #16, where the tests `CI / test (3.9, ubuntu-latest, true) (pull_request)` are failing to run.  The other runs are successful.

#### Problem description

This failure is problematic, because it means the unit tests aren't run at all and we don't know if they would be successful or not.

#### Expected Output

pytroll-cspp-runner main should pass with experimental and non-experimental dependencies.

#### Actual Result, Traceback if applicable

```
Collecting git+https://github.com/mapbox/rasterio
  Cloning https://github.com/mapbox/rasterio to /tmp/pip-req-build-n4oacao2
  Running command git clone --filter=blob:none --quiet https://github.com/mapbox/rasterio /tmp/pip-req-build-n4oacao2
  Resolved https://github.com/mapbox/rasterio to commit fe02e31c48d5dcfeabc3feed2861a28d87af9400
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [2 lines of output]
      WARNING:root:Failed to get options via gdal-config: [Errno 2] No such file or directory: 'gdal-config'
      ERROR: A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
Error: Process completed with exit code 1.
```

#### Versions of Python, package at hand and relevant dependencies

pytroll-cspp-runner main, Python 3.9, experimental dependencies (GitHub main)

--- a/continuous_integration/environment.yaml
+++ b/continuous_integration/environment.yaml
@@ -19,6 +19,7 @@ dependencies:
   - pytest-cov
   - appdirs
   - h5py
+  - gdal
   - pip
   - pip:
     - trollsift

